packages/xo-server/src/xo-mixins/backups-ng/index.mjs

```diff
@@ -484,6 +484,21 @@ export default class BackupNg {
         })
       }
       return result.id
+    } catch (error) {
+      if (
+        // xxhash is the new form consistency hashing in CH 8.1 which uses a faster,
+        // more efficient hashing algorithm to generate the consistency checks
+        // in order to support larger files without the consistency checking process taking an incredibly long time
+        error.code === 'IMPORT_ERROR' &&
+        error.params?.some(
```

Comment:
if `error.code === 'IMPORT_ERROR'`, `errors.params` is not always defined ?

Reply:
I did a search in the xapi code, it looks like it's still defined but I am not sure.